runner/src/main.rs

```diff
@@ -63,11 +67,20 @@ pub enum CommandResult {
     Failure(anyhow::Error),
 }
 
+// TODO : Enable metrics/tracing for load generator command
+// Metrics and tracing have been disabled for load generator due to memory issues.
+// Memory grows in the runner when this is enabled not making it live long enough to finish the load generation
 #[tokio::main]
 async fn main() -> Result<()> {
     let args = Cli::parse();
-    telemetry::init_tracing(Some(args.otlp_endpoint.clone())).await?;
-    let metrics_controller = telemetry::init_metrics_otlp(args.otlp_endpoint.clone()).await?;
+    if !matches!(args.command, Command::GenerateLoad(_)) {
```

Comment:
might be nice to do metrics/tracing in one statement... and using match might be simpler but it's pretty much same same but different

```rust
let metrics_controller =  match &args.command {
        Command::GenerateLoad(_) => None,
    _ => {
        telemetry::init_tracing(Some(args.otlp_endpoint.clone())).await?;
        Some(telemetry::init_metrics_otlp(args.otlp_endpoint.clone()).await?)
    }};
```